WcaOnRails/app/views/shared/_error_messages.html.erb

```diff
@@ -10,7 +10,7 @@
             <% else %>
               <strong><%= f.object.class.human_attribute_name(attribute) %></strong>
             <% end %>
-            <%= msg %>
+            <%= msg.html_safe %>
```

Comment:
Had to add this to show a link, but is this safe for us? Do any message ever show something coming from a user input that's not validated?

Reply:
If you rename the translation key to `unique_html` instead of `unique`, it should works without having to call `html_safe`.
It's fine to call it though, all we display comes from our locale files.

Reply:
Oooh this is much better. Changed to `unique_html`. Of course this means Jonatan's Internationalize will see it as a new key to translate, but maybe that's good, so all locales can benefit from the new version in the meantime, even if it's in English.

Reply:
That's fine, it's actually recommended to change the key when changing the arguments (although in that way - adding arguments - it's fine; whereas not passing an argument to a string which expects it is an error).